src/alert/alert.js

```diff
@@ -0,0 +1,17 @@
+import jQuery from 'jquery';
+import template from './alert.pug';
+
+export default function alert(message) {
+	const html = template({message : message});
+	const component = jQuery(html);
+
+	component.appendTo('body');
+
```

Comment:
There is no need to leave white space between commands like this (it's fine between assignments and commands).

This reads better:

```js
export default function alert(message) {
  const html = template({message : message});
  const component = jQuery(html);

  component.appendTo('body');
  component.modal('show');
  component.find('button').click(function() {
    component.modal('hide');
    component.remove();
  });
}
```